pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -23,64 +24,34 @@ def __init__(self):
+        is also initialised and associated parameters are called from the
+        parameters class.
 
         """
-        # Build infection state transition matrix and set as parameter
+        # Instantiate state transition matrix
         matrix_object = StateTransitionMatrix()
         self.state_transition_matrix =\
             matrix_object.create_state_transition_matrix()
-        pe.Parameters.instance().state_transition_matrix =\
-            self.state_transition_matrix
-
         self.number_of_states = len(InfectionStatus)
         assert self.state_transition_matrix.shape ==\
             (self.number_of_states, self.number_of_states),\
             'Matrix dimensions must match number of infection states'
 
-    def _update_time_to_status_change(self, person: Person, time: float):
-        """Assigns time until next infection status update,
-        given as a random integer between 1 and 10. Used
-        for persons with infection statuses that have no transition/
-        latent time implemented yet - temporary function.
-
-        Parameters
-        ----------
-        Person : Person
-            Instance with infection status attributes
-        time: float
-            Current simulation time
+        # Instantiate transmission time matrix
+        time_matrix_object = TransitionTimeMatrix()
+        self.transition_time_matrix =\
+            time_matrix_object.create_transition_time_matrix()
 
-        """
-        # This is left as a random integer for now but will be made more
-        # complex later.
-        new_time = float(random.randint(1, 10))
-        person.time_of_status_change = time + new_time
-
-    def _set_latent_time(self, person: Person, time: float):
-        """Calculates latency period as calculated in CovidSim,
-        and updates the time_of_status_change for the given
-        Person, given as the time until next infection status
-        for a person who has been set as exposed.
-
-        Parameters
-        ----------
-        Person : Person
-            Instance with infection status attributes
-        time: float
-            Current simulation time
-
-        """
-        latent_period = pe.Parameters.instance().latent_period
-        latent_period_iCDF = pe.Parameters.instance().latent_period_iCDF
-        latent_icdf_object = InverseCdf(latent_period, latent_period_iCDF)
-        latent_time = latent_icdf_object.icdf_choose_exp()
-
-        assert latent_time >= 0.0, 'Negative latent time'
-
-        person.time_of_status_change = time + latent_time
+        # Instantiate parameters to be used in update transition time
+        # method
+        self.latent_to_symptom_delay =\
+            pe.Parameters.instance().latent_to_sympt_delay
+        self.model_time_step = 1 / pe.Parameters.instance().time_steps_per_day
+        self.delay = np.floor(self.latent_to_symptom_delay /
+                              self.model_time_step)
```

Comment:
```suggestion
        self.delay = np.floor(self.latent_to_symptom_delay,
                              self.model_time_step)
```

Reply:
This is a divide, not a backslash. Have changed to make line break before mathematical operator per PEP8

Reply:
Ahh that makes sense sorry - thanks for that!